flixel/FlxCamera.hx

```diff
@@ -526,119 +529,128 @@ class FlxCamera extends FlxBasic
-		
-		while (i < verticesLength)
+		if(FlxG.renderBlit)
 		{
-			tempX = position.x + vertices[i]; 
-			tempY = position.y + vertices[i + 1];
+			if (position == null)
+			{
+				position = FlxPoint.flxPoint1.set(0, 0);
+			}
+			
+			_bounds.set(0, 0, width, height);
+			
+			var verticesLength:Int = vertices.length;
+			var currentVertexPosition:Int = 0;
 			
-			drawVertices[currentVertexPosition++] = tempX;
-			drawVertices[currentVertexPosition++] = tempY;
+			var tempX:Float, tempY:Float;
+			var i:Int = 0;
+			var bounds:FlxRect = FlxRect.flxRect;
+			drawVertices.splice(0, drawVertices.length);
 			
-			if (i == 0)
+			while (i < verticesLength)
 			{
-				bounds.set(tempX, tempY, 0, 0);
+				tempX = position.x + vertices[i]; 
+				tempY = position.y + vertices[i + 1];
+				
+				drawVertices[currentVertexPosition++] = tempX;
+				drawVertices[currentVertexPosition++] = tempY;
+				
+				if (i == 0)
+				{
+					bounds.set(tempX, tempY, 0, 0);
+				}
+				else
+				{
+					FlxDrawTrianglesItem.inflateBounds(bounds, tempX, tempY);
+				}
+				
+				i += 2;
+			}
+			
+			var vis:Bool = _bounds.overlaps(bounds);
+			
+			if (!vis)
+			{
+				drawVertices.splice(drawVertices.length - verticesLength, verticesLength);
 			}
 			else
 			{
-				FlxDrawTrianglesItem.inflateBounds(bounds, tempX, tempY);
+				trianglesSprite.graphics.clear();
+				trianglesSprite.graphics.beginBitmapFill(graphic.bitmap, null, false, smoothing);
+				trianglesSprite.graphics.drawTriangles(drawVertices, indices, uvtData);
+				trianglesSprite.graphics.endFill();
+				buffer.draw(trianglesSprite);
+				#if !FLX_NO_DEBUG
+				if (FlxG.debugger.drawDebug)
+				{
+					var gfx:Graphics = FlxSpriteUtil.flashGfx;
+					gfx.clear();
+					gfx.lineStyle(1, FlxColor.BLUE, 0.5);
+					gfx.drawTriangles(drawVertices, indices);
+					camera.buffer.draw(FlxSpriteUtil.flashGfxSprite);
+				}
+				#end
 			}
-			
-			i += 2;
-		}
-		
-		var vis:Bool = _bounds.overlaps(bounds);
-		
-		if (!vis)
-		{
-			drawVertices.splice(drawVertices.length - verticesLength, verticesLength);
 		}
 		else
 		{
-			trianglesSprite.graphics.clear();
-			trianglesSprite.graphics.beginBitmapFill(graphic.bitmap, null, false, smoothing);
-			trianglesSprite.graphics.drawTriangles(drawVertices, indices, uvtData);
-			trianglesSprite.graphics.endFill();
-			buffer.draw(trianglesSprite);
-			#if !FLX_NO_DEBUG
-			if (FlxG.debugger.drawDebug)
-			{
-				var gfx:Graphics = FlxSpriteUtil.flashGfx;
-				gfx.clear();
-				gfx.lineStyle(1, FlxColor.BLUE, 0.5);
-				gfx.drawTriangles(drawVertices, indices);
-				camera.buffer.draw(FlxSpriteUtil.flashGfxSprite);
-			}
-			#end
-		}	
+			_bounds.set(0, 0, width, height);
+			var isColored:Bool = (colors != null && colors.length != 0);
+			var drawItem:FlxDrawTrianglesItem = startTrianglesBatch(graphic, smoothing, isColored, blend);
+			drawItem.addTriangles(vertices, indices, uvtData, colors, position, _bounds);
+		}
 	}
-#end
+
+//else if FlxG.renderBlit
```

Comment:
I'm not sure.. These sections are so spread out that I doubt that would be maintained...